File: task-03/Go/Sub-Task2.go
package main

import (
    "io/ioutil"
    "log"
)

func main() {
    // Read the contents of input.txt
    input, err := ioutil.ReadFile("input.txt")
    if err != nil {
        log.Fatal(err)
    }

    // Write the contents to output.txt
    err = ioutil.WriteFile("output.txt", input, 0644)
    if err != nil {
        log.Fatal(err)
    }
}

File: task-03/Javascript/Sub-Task2.js
const fs = require('fs');

// Function to copy content from input.txt to output.txt
function copyFile() {
    // Read the content from input.txt
    fs.readFile('input.txt', 'utf8', (err, data) => {
        if (err) {
            console.error('Error reading input.txt:', err);
            return;
        }

        // Write the content to output.txt
        fs.writeFile('output.txt', data, (err) => {
            if (err) {
                console.error('Error writing to output.txt:', err);
                return;
            }

            console.log('File copied successfully.');
        });
    });
}

// Call the function
copyFile();
